src/myuplink/models.py

```diff
@@ -138,25 +198,50 @@ def id(self) -> str:
         return self.raw_data["id"]
 
     @property
-    def productName(self) -> str:
+    def product_name(self) -> str:
         """Return the name of the device product."""
         return self.raw_data["product"]["name"]
 
     @property
-    def productSerialNumber(self) -> str:
+    def productName(self) -> str:
+        """Return the name of the device product. Deprecated 2024-04-30."""
+        return self.productName
```

Comment:
I think there is a typo here - i did a quick test with the example1.py file.

Think it should be:  `return self.product_name`